icepyx/core/variables.py

```diff
@@ -97,16 +119,14 @@ def avail(self, options=False, internal=False):
         .
         'quality_assessment/gt3r/signal_selection_source_fraction_3']
         """
-        # if hasattr(self, '_avail'):
-        #         return self._avail
-        # else:
+
         if not hasattr(self, "_avail") or self._avail == None:
-            if self._vartype == "order":
+            if not hasattr(self, 'path'):
                 self._avail = is2ref._get_custom_options(
-                    self.session, self.product, self._version
+                    self.session, self.product, self.version
```

Comment:
This line made me notice how internally inconsistent we are with having (e.g.) `_version` vs `version` across modules (something you previously mentioned, @rwegener2). Since they're generally attributes the user shouldn't change, should we make them private and give them property accessors within the variables module?

Reply:
I think that's a great idea!